docs/en/latest/getting-started.md

```diff
@@ -92,29 +98,31 @@ If you already have Apache APISIX installed, please skip Step 1, and go to [Step
 
 ## Step 1: Install Apache APISIX
 
-Thanks to Docker, we can start Apache APISIX and enable it by enabling [Admin API](./admin-api.md).
+You can check out [Building Apache APISIX](./how-to-build.md) for different installation methods.
+
+To get started quickly, we will install Apache APISIX with Docker and enable the [Admin API](./admin-api.md).
 
 ```bash
-# Download the Docker image of Apache APISIX
+# Download the docker-compose file of Apache APISIX
 git clone https://github.com/apache/apisix-docker.git
-# Switch the current directory to the apisix-docker/example path
+# Switch the current directory to the apisix-docker/example
 cd apisix-docker/example
-# Run the docker-compose command to install Apache APISIX
+# Start Apache APISIX with docker-compose
 docker-compose -p docker-apisix up -d
 ```
 
-> Apache APISIX has already supported ARM64 architecture. For ARM64 users, please use `docker-compose -p docker-apisix -f docker-compose-arm64.yml up -d` instead in the last step.
+> Apache APISIX already supports ARM64 architecture. To run Apache APISIX on ARM64, run: `docker-compose -p docker-apisix -f docker-compose-arm64.yml up -d` instead of the last step above.
 
-It will take some time to download all required files, please be patient.
+Please remain patient as it will take some time to download the files and spin up the containers.
 
-Once the download is complete, execute the `curl` command on the host running Docker to access the Admin API, and determine if Apache APISIX was successfully started based on the returned data.
+Once Apache APISIX is running, you can use `curl` to access the Admin API. You can also check if Apache APISIX is running properly by running this command and checking the response.
 
 ```bash
-# Note: Please execute the curl command on the host where you are running Docker.
+# Execute in your host machine (machine running Docker)
```

Comment:
Would be better to add a**on** or **in** between `running` and `Docker` in the end of this sentence.
```suggestion
# Execute in your host machine (machine running Docker)
```

Reply:
We should be running the command on the machine that is running Docker right and **not** on inside the Docker container right?

Reply:
OK cool. We can skip this comment.